api/src/main/java/io/opencensus/trace/SpanBuilder.java

```diff
@@ -164,12 +157,13 @@ public SpanBuilder setRecordEvents(boolean recordEvents) {
    * <p>This is useful when {@link Tracer#spanBuilder(String)} is used and the newly created {@code
    * Span} needs to be decoupled from the parent {@code Span}.
```

Comment:
side note.. this sounds like it is explaining why, but is really another way of saying what. Can you give an example where this would be used?